### PR TITLE
Adding toggle method to pedal mock

### PR DESCRIPTION
**Pull request by ethanfowler**
## Proposed changes

Adding toggle method to pedal mock, useful for uses of the library (e.g. my recent testing)

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [x] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [x] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: sr_pedal/src/sr_pedal/sr_triple_pedal_mock.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # -*- coding: latin-1 -*-
 
-# Copyright 2020 Shadow Robot Company Ltd.
+# Copyright 2020-2022 Shadow Robot Company Ltd.
```
Comment (bdenoun) [inline sr_pedal/src/sr_pedal/sr_triple_pedal_mock.py:5]: Quick check, has this file been modified in 2021?

**Review by bdenoun — APPROVED** [review]

File: sr_pedal/src/sr_pedal/sr_triple_pedal_mock.py [hunk 2]
```diff
@@ -104,6 +111,16 @@ def set_status(self, connected=None, left_pressed=None, middle_pressed=None, rig
                 rospy.loginfo("Mock right pedal now released.")
         self._lock.release()
 
+    def toggle_status(self, status):
+        if status == "connected":
+            self._status.connected = not self._status.connected
+        if status == "left":
```
Comment (maxzieba) [inline sr_pedal/src/sr_pedal/sr_triple_pedal_mock.py:117]: elifs?

Comment (maxzieba) [inline sr_pedal/src/sr_pedal/sr_triple_pedal_mock.py]: elifs?

**Review by maxzieba — APPROVED** [review]

File: sr_pedal/src/sr_pedal/sr_triple_pedal_mock.py (hunk 1 above)
Comment (ethanfowler) [reply sr_pedal/src/sr_pedal/sr_triple_pedal_mock.py:5]: Yup https://github.com/shadow-robot/sr_teleop_devices/commits/be0c876b5d9c0e4b806c10f4d25dbcfd3650a43c/sr_pedal/src/sr_pedal/sr_triple_pedal_mock.py

File: sr_pedal/src/sr_pedal/sr_triple_pedal_mock.py (hunk 2 above)
Comment (ethanfowler) [reply sr_pedal/src/sr_pedal/sr_triple_pedal_mock.py]: Whoops, quite right, fixed elsewhere too in https://github.com/shadow-robot/sr_teleop_devices/pull/52/commits/34820e8e14811b298d6b42d8df54b400d78b1d21

**Pull request merged**